chainer/links/connection/zoneoutlstm.py

```diff
@@ -0,0 +1,113 @@
+import numpy
+
+import chainer
+from chainer.functions.activation import sigmoid
+from chainer.functions.activation import tanh
+from chainer.functions.array import reshape
+from chainer.functions.array import split_axis
+from chainer.functions.noise import zoneout
+
+from chainer import link
+from chainer.links.connection import linear
+from chainer import variable
+
+
+class ZoneoutLSTM(link.Chain):
```

Comment:
Could you change the name of the chain to "StatefulZoneoutLSTM" to follow the naming convention of other RNN units?